examples/reference/chat/ChatFeed.ipynb

```diff
@@ -85,7 +85,7 @@
    "outputs": [],
    "source": [
     "chat_feed = pn.chat.ChatFeed()\n",
-    "chat_feed"
+    "chat_feed.show()"
```

Comment:
`.show()` in a notebook? I guess that you want to remove that.

Reply:
Thanks for catching that; I can clean this up.